core/src/main/java/tc/oc/pgm/scoreboard/SidebarMatchModule.java

```diff
@@ -489,21 +490,60 @@ public int compare(Goal a, Goal b) {
                 }
               });
 
+          boolean inline = false;
+          String altText = " ";
+          int index = 1;
           for (Goal goal : sortedWools) {
             if (goal instanceof MonumentWool && goal.isVisible()) {
               MonumentWool wool = (MonumentWool) goal;
+              if (wool.getDefinition().isInline() == true) {
+                inline = true;
+              }
               if (!firstWool) {
-                woolText += "   ";
+                if (sortedWools.size() > 5) {
```

Comment:
Again, instead of basing this on amount of wools, base it off of if the characters will fit